races/android.md

```diff
@@ -1,3 +1,87 @@
 ---
 wrapper: "../../../js/components/wiki/StubArticle"
 ---
+
+
+## Android
+<br><br>
+#### Description<br>
+The androids are robotic and cybernetic units designed for a variety of military and hazardous civilian tasks.  They have limited intelligence but are capable of immense physical strength.  Unlike organic races, androids have no mana.  They are not recommended for inexperienced players.<br><br>
+
+|Starting Stats| |
+|:---|:---|
+|Str | 20 |
+|Int | 7 |
+|Wil | 7 |
+|Con | 20 |
+|Dex | 5 |
+|Chr | 3 |
+
+ <br>
+
+
+
+
+Hitpoints	100 <br>
+Mana		0<br> 
+Fatigue		61<br> 
+Free Points	0<br> 
+<br>
+
+|Race Resistances| |
+|:---|:---|
+| Asphyxiation | *Completely resistant* |
+| Cold | Somewhat susceptible |
+| Disease| *Completely resistant* |
+| Edged | Somewhat resistant |
+| Electric | **Devastatingly susceptible** | 
+| Energy | Moderately susceptible |
+| Fire | Moderately resistant |
+| Magic Electric | **Devastatingly susceptible** |
+| Magic Fire | Moderately resistant |
+| Piercing | Moderately resistant |
+| Poison | *Completely resistant* |
+| Psychic | *Completely resistant* |
+
+---
```

Comment:
I personally am not a fan of a lot of horizontal rules unless you have really long articles with  major subsections